src/main/java/org/spongepowered/common/mixin/core/command/MixinCommandBase.java

```diff
@@ -43,4 +50,15 @@ public void setExpandedSelector(boolean expandedSelector) {
         this.expandedSelector = expandedSelector;
     }
 
+    @Override
+    public void updateNamespacedAlias(String ownerId) {
+        this.namespacedAlias = ownerId + ":" + getName();
+    }
+
+    @Redirect(method = "checkPermission(Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/command/ICommandSender;)Z",
```

Comment:
Yeah, fair point. I did this whilst trying to get it to work before, didn't think about it after. Will test and then commit.